Sources/Hummingbird/Application.swift

```diff
@@ -147,11 +168,25 @@ public final class HBApplication: HBExtensible {
         try startError.map { throw $0 }
     }
 
-    /// wait while server is running
+    /// Wait until server has stopped running
+    ///
+    /// This function can only be called from a non async context as it stalls
+    /// the current thread waiting for the application to finish
+    @available(*, noasync, message: "Use HBApplication.asyncWait instead.")
```

Comment:
Not sure if this is a breaking change, but calling `wait` in an asynchronous context is totally borked anyway, I think we should allow this through.